gradle-jdks-excavator-configurations/src/main/resources/install-jdks.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+set -eux
+
+die () {
+    echo
+    echo "$*"
+    echo
+    rm -rf "$tmp_work_dir"
+    exit 1
+} >&2
+
+read_value() {
+  if [ ! -f "$1" ]; then
+    die "ERROR: $1 not found, aborting Gradle JDK installation"
+  fi
+  read -r value < "$1" || die "ERROR: Unable to read value from $1. Make sure the file ends with a newline."
+  echo "$value"
+}
+
+GRADLE_JDKS_DIR=$1
+CERTS_DIR=$2
+
+if [[ -z "$CERTS_DIR" || -z "$GRADLE_JDKS_DIR" ]]; then
+  die "ERROR: Missing argument CERTS_DIR"
+fi
+
+GRADLE_USER_HOME=${GRADLE_USER_HOME:-"$HOME"/.gradle}
+GRADLE_JDKS_HOME="$GRADLE_USER_HOME"/gradle-jdks
+mkdir -p "$GRADLE_JDKS_HOME"
+
+# OS specific support; same as gradle-jdks:com.palantir.gradle.jdks.setup.common.CurrentOs.java
+case "$( uname )" in                          #(
+  Linux* )          os_name="linux"  ;;       #(
+  Darwin* )         os_name="macos"  ;;       #(
+  * )               die "ERROR Unsupported OS: $( uname )" ;;
+esac
+
+if [ "$os_name" = "linux" ]; then
+    ldd_output=$(ldd --version 2>&1 || true)
+    if echo "$ldd_output" | grep -qi glibc; then
+       os_name="linux-glibc"
+    elif echo "$ldd_output" | grep -qi "gnu libc"; then
+           os_name="linux-glibc"
+    elif echo "$ldd_output" | grep -qi musl; then
+      os_name="linux-musl"
+    else
+      die "Unable to determine glibc or musl based Linux distribution: ldd_output: $ldd_output"
+    fi
+fi
+
+# Arch specific support, see: gradle-jdks:com.palantir.gradle.jdks.setup.common.CurrentArch.java
+case "$(uname -m)" in                         #(
```

Comment:
It feels to me like a lot of this logic should be shared with `gradle-jdks-setup.sh`? I'm worried that copying so much of the logic like this would mean they get out of sync.